pkg/memtier/prompt_test.go

```diff
@@ -42,6 +42,9 @@ func FuzzPrompt(f *testing.F) {
 	pagesCommonArgs := fmt.Sprintf(" -pid %d ", os.Getpid())
 	testcases := []string{
 		"help",
+		"log -i \"my info\" -d 'my debug' -e my\\ error",
```

Comment:
Note that in your test the `memtierd-command` function internally uses `echo -e '...'` to send the command to the memtierd process. If the command (`...`) contains a single quote, like `log -d 'my debug'`, this echo command becomes:
```
echo -e 'log -d 'my debug''
```
... and this kind of quoting is handled by bash similarly to the argument list
```
["echo", "-e", "log -d ", "my", "debug"]
```
and that is why memtierd prompt will not even get the single quote. That is, the problem you saw is the problem of the memtierd-command function, and not of the memtierd prompt. If you run
```
memtierd -prompt
memtierd> log -i 'my info'
```
...you will notice that single quote works just fine, as now memtierd-prompt actually gets it.

It would be nice to write the memtierd-command function so that single quotes will not be lost when gluing together the command to be executed on vm. This could be done by piping the parameter to a file on the vm and then redirecting contents of that file to socat. But that's not related to this PR, so I won't do it here.